src/mame/skeleton/pcksurfer.cpp

```diff
@@ -0,0 +1,91 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/***************************************************************************************************
+
+Datawind Pocket Surfer.
+
+Pre-smartphone era pocket internet terminal.
+
+https://web.archive.org/web/20060221032558/http://www.datawind.com/
+https://web.archive.org/web/20060221032551/http://www.datawind.com/specs.html
+https://uk.pcmag.com/first-looks/7579/datawind-pocketsurfer
+
+CPU Sharp LH79524-NOF (package BGA-208).
+
+Tear down of a production unit: https://www.youtube.com/watch?v=lQi8veu3ugU
+
+PCB:
+                                 _____
+  _______________________________|   |__  _______________
+ _|               _____________  |USB|  |_|              \_____________________
+_|    ________    |            |                                _______        |
+|     |DW-    |   |            |                                |STB5610       |
+|     |SBTM1  |   |            |                                |      |       |
+|     |Bluetooth  |   SIM      |                                |______|       |
+|     |_______|   |  SOCKET    | ___         _____________            _______  |
+|   ___________   |   AND      ||___|  ___   | IS42S16400B|           |      | |
+|   |U4 Flash  |  |  COMMS     |       |  |  | DRAM       | ________  |IS42S16400B
+|   |          |  |            | LVX457->_|  |____________| |SHARP  | |DRAM  | |
+|   |__________|  |____________|       ___                  |LH79524| |      | |
+|                           ___        |  |                 |NOF    | |      | |
+|                    LCX125->__|       |_<-LCX125           |_______| |______| |
+|______________________________________________________________________________|
+
+Chip labeled "31314 3A05U 511AD" inside the DW-SBTM1 module.
+
+
+    TODO:
+    - Everything
+*/
+
+#include "emu.h"
+
+#include "cpu/arm7/arm7.h"
+#include "cpu/arm7/arm7core.h"
```

Comment:
You shouldn’t need to include internal headers.

Reply:
Done, thanks!!